### ISSUE TITLE
FNX3-17004 ⁃ [Bug][a11y] '+' button from Add new collection read as Close button using Talkback

### ISSUE THREAD

**Issue by sv-sdeiac** [mozilla-mobile/fenix#3424]
## Prerequisites
You have Talkback option enabled from your Android settings.

## Steps to reproduce
1. Open Firefox preview;
1. Open a page;
1. Tap 3dots menu;
1. Tap on 'Save to Collection' button;
1. Make Talkback to read the '+' button from Add new collection.

### Expected behavior
Talkback read correctly the '+' button.

### Actual behavior
Talkback read the '+' button as Close button.

### Device information
* Android device: LG G7fit (Android 8.1).
* Fenix version: 1.0.1924.

**Comment by JorgeDLS** [comment]
Hi, I'd like to grab this one, but I have a doubt:
The '+' button here acts as a complement to the text it has at its side (Add new collection) so I'm not sure if it's best for Talkback to read the + button as "Add new Collection" or it would be better to omit this item from being read by Talkback by removing its contentDescription attribute.

If we take the #1 approach, that would result in talkback reading two "Add new Collection" which I think could be misleading, that's why I propose removing contentDescription from that ImageButton.

What do you folks think?

**Comment by ekager** [comment]
I think the content description for that button should change with the mode of collection creation. So in `CollectionCreationUIView` when we change the drawable for `bottom_bar_icon_button` , we should also add/remove the content description. It should read "close" but I agree with you we don't want duplicate descriptions read for "Add new Collection".

**Comment by sv-sdeiac** [comment]
Hello, in my opinion, is that we should make a translation more accurate of the '+' button. 
It is not a good idea to remove the name of a button from Talkback.  
Maybe we can find a description to be different from the "Add new Collection". It can be just 'Add button' or eventually if we can't find a good Idea for that button, we can connect the '+' with the 'Add new collection' text to be read in the same time.

**Comment by JorgeDLS** [comment]
I fully understand your concerns @sv-sdeiac, in fact, I'll try to explain better the situation and why I think removing the `ImageButton` from Talkback is good for this particular case:

When adding a new collection, there's a "Bottom bar" that's made from an `ImageButton` and a `TextView`. The whole BottomBar is the touchable/clickable/readable by talkback element, so in the actual implementation, talkback reads the BottomBar (Add New Collection), and it also reads the `ImageButton` with the plus sign. If we follow [Mozilla's Accessibility Guide](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md#cut-out-redundant-steps-with-androidimportantforaccessibilityno), they suggest removing redundant steps like this one.

So, long story short, there's no functionality being lost here, in fact, a redundant step is removed, which is good for the UX.

**Comment by AndiAJ** [comment]
Hi, v**erified as fixed** on **1.3.0-rc.1** using a Huawei Media Pad M3 Lite 10 (Android 7)

**Comment by softvision-miralobontiu** [comment]
Also, verified on RC 1.3.0-rc.3 with Xiaomi Mi8 Lite (Android 9).
